rsts/administrator/install/authentication.rst

```diff
@@ -0,0 +1,88 @@
+.. _install-authentication:
+
+#################
+Authentication
+#################
+
+Flyte Admin ships with a canonical implementation of OAuth2, integrating seamlessly into an organization's existing identity provider.  At Lyft, we use Okta as the IDP, but if you have issues integrating with another implementation of the OAuth server, please open an issue.
+
+***********************
+Components
+***********************
+
+While the most obvious interaction with the Flyte control plane is through the web based UI, there are other critical components of Flyte that also need to be considered. These components should be thought of as third-party services even though the Flyte codebase provides them.
+
+Flyte CLI
+=========
+Principal amongst these is the Flyte CLI. This is the command-line entrypoint to Flyte Admin and is used by both administrators and users more comfortable in the command line, or are running in a headless OS.
+
+The IDP application corresponding to the CLI will need to support PKCE.
+
+
+Direct Client Access
+====================
+The gRPC client provided by the Flyte IDL, or direct calls to the HTTP endpoints on Admin from within a running script are ways that we have seen users hit the control plane directly.  We generally discourage this behavior as it leads to a possible self-imposed DOS vector, as they are generally made from within a running workflow itself. For instance, a Flyte task can fetch the definition for a launch plan associated with a completely different workflow, and then launch an execution of it. This is not the correct way to launch one workflow from another but for the time being remains possible.
+
+
+******
+Flow
+******
+
+Flyte Admin authentication is implemented using the authorization code flow.
+
+Flyte UI Flow
```

Comment:
Can we move these images into our gh-pages branch

Reply:
Yes, I will do that.